tests/backup_helper.go

```diff
@@ -9506,3 +9506,465 @@ func GetCloudsnapStatus() (map[string]CloudsnapStatus, error) {
 	}
 	return statusMap, nil
 }
+
+// CreatePartialBackupWithVscMapping creates backup with Vsc mapping and checks for partial success
+func CreatePartialBackupWithVscMapping(backupName string, clusterName string, bLocation string, bLocationUID string,
+	namespaces []string, labelSelectors map[string]string, orgID string, uid string, preRuleName string,
+	preRuleUid string, postRuleName string, postRuleUid string, ctx context1.Context, provisionerVolumeSnapshotClassMap map[string]string, forceKdmp bool) error {
```

Comment:
Why are we taking `provisionerVolumeSnapshotClassMap map[string]string, forceKdmp bool` but not using them?

Reply:
Something messed up while rebasing, corrected it. Thanks @mkoppal-px .